landoapi/transplant_client.py

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+import requests
+import requests_mock
+
+
+class TransplantClient:
+    """ A class to interface with Transplant's API. """
+
+    def __init__(self):
+        self.api_url = os.getenv('TRANSPLANT_URL') + '/api'
+
+    @requests_mock.mock()
+    def land(
+        self,
+        ldap_username,
+        tree,
+        patch,
+        destination,
+        push_bookmark,
+        pingback_url,
+        request,  # mocked requests
+        rev='',
+        commit_descriptions=''
+    ):
+        """ Sends a push request to Transplant API to land a revision.
+
+        Returns request_id received from Transplant API.
+        """
+        # Connect to stubbed Transplant service
+        request.post(
+            self.api_url + '/autoland',
+            text=json.dumps({
+                'request_id': 1
+            }),
+            status_code=202
+        )
+
+        result = self._POST(
+            '/autoland', {
```

Comment:
Some of these values aren't needed, e.g. 'rev', 'commit descriptions', etc. See the second page of https://docs.google.com/document/d/1cRIdpVqYcn0Mle7vyjJePNw5TcFoZ20N42dLVy5xVEk/edit#heading=h.rfxbgux0479w for details.

For the ones that aren't needed, lets remove them as arguments to the function. Then others that will never change like `pingback_url`, lets set them here are remove them as arguments to the function.